css/main.css

```diff
@@ -13,13 +12,13 @@ h2 {
   float: left;
   width: 100%;
   height: 100%;
-  text-shadow: 1px 1px 2px rgba(0,0,0,.3);
-  letter-spacing: .05em;
+  text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.3);
+  letter-spacing: 0.05em;
 }
 
 form {
   width: 80vw;
-  height: 80vh;
+  height: 45vh;
```

Comment:
I worry that this might hide some of the work you do on your particular page in the future. If you plan on creating a drop-down list in your shipping or shopping cart page, this will probably cause some problems.

Reply:
the only reason I changed it is because it wasn't formatting correctly at the 80vh. I can make it 45vh on my page specifically so that it won't affect other pages indirectly.